caffeine/src/main/java/com/percolate/caffeine/DialogUtils.java

```diff
@@ -28,15 +31,16 @@
      * @return AlertDialog that is being displayed.
      */
     public static AlertDialog quickDialog(final Activity context, final String message) {
+        if(dialog != null && dialog.isShowing()) return null;
```

Comment:
@brentwatson You are implying returning `<dialog>` with the updated textview text, correct?